web/token.go

```diff
@@ -0,0 +1,101 @@
+package web
+
+import (
+	"encoding/base32"
+	"encoding/gob"
+	"os"
+	"path/filepath"
+
+	gweb "github.com/G-Node/gin-cli/web"
+	"github.com/G-Node/gin-valid/config"
+	"github.com/G-Node/gin-valid/log"
+)
+
+// saveToken writes a token to disk using the username as filename.
+// The location is defined by config.Dir.Tokens.
+func saveToken(ut gweb.UserToken) error {
+	cfg := config.Read()
+	filename := filepath.Join(cfg.Dir.Tokens, ut.Username)
+	tokenfile, err := os.Create(filename)
+	defer tokenfile.Close()
+	if err != nil {
+		return err
+	}
+	encoder := gob.NewEncoder(tokenfile)
+	return encoder.Encode(ut)
+}
+
+// loadUserToken reads a token from disk using the username as filename.
+// The location is defined by config.Dir.Tokens.
+func loadUserToken(username string) (gweb.UserToken, error) {
```

Comment:
Does it makes sense to use the name `getTokenByUser` analogous to `getTokenBySession`?